### PR TITLE
Change the server runtime temp directory to ~/.jasmine/tmp

### PR DESCRIPTION
**Pull request by tmkasun**

### CODE REVIEW

File: src/util/Utils.cpp [hunk 1]
```diff
@@ -215,6 +217,20 @@ std::string Utils::getFileName(std::string filePath) {
     return filename;
 }
 
+/*
+ * Get the current users(calle of the program) home directory from the $HOME environment variable,
```
Comment (miyurud) [inline src/util/Utils.cpp]: Change "Get the current users(calle of the program)" to "Get the current user's (caller of the program)"

File: src/util/Utils.h [hunk 2]
```diff
@@ -50,6 +50,8 @@ class Utils
     std::string getFileName(std::string filePath);
 
     int getFileSize(std::string filePath);
+    // Static method to get running users home directory
```
Comment (miyurud) [inline src/util/Utils.h]: change users to user's

Comment (tmkasun) [reply src/util/Utils.h]: Fixed with 
532a9a7 commit

File: src/util/Utils.cpp (hunk 1 above)
Comment (tmkasun) [reply src/util/Utils.cpp]: Fixed in 532a9a7 commit

**Pull request merged**